src/usr/share/netcheck/netcheck.py

```diff
@@ -88,8 +87,8 @@ def _dns_query(self, network, nameserver, query_name):
 
         except dns.resolver.NXDOMAIN as detail:
             # Could be either a config error or malicious DNS
-            self.logger.error('DNS query for %s from nameserver %s on network %s was successful, ' +
-                'but the provided domain was not found. %s: %s' %
+            self.logger.error('DNS query for %s from nameserver %s on network %s was successful,'
```

Comment:
I don't think this line complies with PEP 8.

Reply:
PEP 8 does not specify how to break up long string literals, but this style allows us to use formatting arguments in a reasonable way.

Reply:
Sorry, I was referring to the indentation. Removing the concatenation is fine.